.nvmrc

```
@@ -0,0 +1 @@
+22.13.0
```

nit:

```suggestion
22.13.0

```

From the nvm readme:

> The contents of a `.nvmrc` file **must** contain precisely one `<version>` (as described by `nvm --help`) followed by a newline.

`nvm install` works fine for me without the newline, but we do have it in all other project files anyway 🤷‍♂️ 

Also a question: shouldn't this version match the one in the `engines` field? We're saying we support `22.0.0` as the minimum version, but nvm users working on this project will be using `22.13.0` so they could end up adding code that doesn't work on `22.0.0` and not noticing it.

I think this is a similar point to @dsanders11's on `test.yml` about testing the lowest supported version - we should pick one of `22.0.0` / `22.13.0` and use it everywhere.

---

I like the idea of having it on the lower bound so that we ensure that development happens on the minimum `engines` version. :)